contracts/Core/RewardManager.sol

```diff
@@ -133,6 +133,7 @@ contract RewardManager is Initializable, ACL, Constants, IRewardManager {
         if (mediansLastEpoch.length == 0) return;
         uint64 penalty = 0;
         for (uint8 i = 0; i < mediansLastEpoch.length; i++) {
+            // slither-disable-next-line calls-loop
             uint48 voteValueLastEpoch = voteManager.getVoteValue(i, stakerId);
             // uint32 voteWeightLastEpoch = voteManager.getVoteWeight(thisStaker.id, i);
```

Comment:
?

Reply:
this is for call inside loop getVoteValue is being called